### PR TITLE
230329 백자민 제출

### PR DESCRIPTION
**Pull request by sdflksjf2683**
✅ 프로그래머스: 더 맵게(https://school.programmers.co.kr/learn/courses/30/lessons/42626)
✅ SWEA: 키 높이(https://swexpertacademy.com/main/code/problem/problemDetail.do?contestProbId=AWXQsLWKd5cDFAUo)
✅ SWEA: 벽돌 깨기(https://swexpertacademy.com/main/code/problem/problemDetail.do?contestProbId=AWXRQm6qfL0DFAUo)

### CODE REVIEW

File: 230429/백자민/SWEA_5656_백자민.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                     
+                    q.offer(new Point(ni,nj,map[ni][nj]));
+                    map[ni][nj] = 0;
+                    visit[ni][nj] = true;
+                }
+            }
+        }
+    }
+     
+    static void perm(int cnt) {
```
Comment (seunghee114) [inline 230429/백자민/SWEA_5656_백자민.java:85]: 순열은 꼭 해야져

File: 230429/백자민/SWEA_5643_백자민.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+ * */
+ 
+public class SWEA_5643_백자민 {
+     
+    static int N,M, cnt;
+     
+    static ArrayList<Integer>[] front, back;
+     
+    static int count(int num) {
+        Queue<Integer> que = new LinkedList<Integer>();
```
Comment (seunghee114) [inline 230429/백자민/SWEA_5643_백자민.java:24]: BFS 좋아영

**Review by seunghee114 — APPROVED** [review]

File: 230429/백자민/SWEA_5656_백자민.java [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        for(int i=0;i<H;i++) {
+            for(int j=0;j<W;j++) {
+                if(map[i][j]!=0)
+                    cnt++;
+            }
+        }
+        return cnt;
+    }
+     
+    static int[][] deepcopy(int[][] origin) {
```
Comment (rin-k645) [inline 230429/백자민/SWEA_5656_백자민.java:144]: deepcopy 오랜만이에요

**Review by rin-k645 — APPROVED** [review]

File: 230429/백자민/SWEA_5643_백자민.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        int T = Integer.parseInt(br.readLine());
+        for(int t=1;t<=T;t++) {
+            N = Integer.parseInt(br.readLine());
+            M = Integer.parseInt(br.readLine());
+             
+            front = new ArrayList[N+1];
+            back = new ArrayList[N+1];
+             
+            for(int n=0;n<=N;n++) { //초기화 
+                front[n] = new ArrayList<Integer>();
```
Comment (rin-k645) [inline 230429/백자민/SWEA_5643_백자민.java:66]: 배열 두 개 써야하는거!! 좋아요

**Review by rin-k645 — APPROVED** [review]

**Pull request merged**